homeassistant/components/overkiz/cover.py

```diff
@@ -24,7 +24,7 @@
 )
 
 from .const import DOMAIN
-from .tahoma_entity import TahomaEntity
+from .overkiz_entity import OverkizEntity
```

Comment:
In a future PR, it would be cleaner to call this `entity.py` since its already in the `overkiz` namespace.